spot_driver/spot_driver/spot_ros2.py

```diff
@@ -844,6 +844,31 @@ def __init__(self, parameter_list: Optional[typing.List[Parameter]] = None, **kw
                 if self.auto_stand.value:
                     self.spot_wrapper.stand()
 
+        # Explicitly get a lease when get_lease_on_action is False
+        # because spot_wrapper does not get a lease at initialization.
+        if self.spot_wrapper is not None and not self.get_lease_on_action.value:
+            self.spot_wrapper.getLease()
```

Comment:
What is the consequence of not adding this change? I think the spirit of this PR could be that if you don´t care about this new functionality (ROS service to generate new lease), then there is no functional change to `spot_ros2`.

Reply:
I can remove this. Not having this just means that anyone commanding through `spot_ros2`, when running `spot_ros2` with `use_take_lease=True` and `get_lease_on_action=False`, will have to ask `spot_ros2` to explicitly create a new lease before it sends any commands to it. But that's fine.